app/services/solidus_bolt/transactions/authorize_service.rb

```diff
@@ -25,7 +26,7 @@ def authorize
 
       def body
         {
-          auto_capture: false,
+          auto_capture: auto_capture,
```

Comment:
Do you mind adding a test for this change?

Reply:
Added a test for this
Check here [745182a](https://github.com/nebulab/solidus_bolt/commit/745182a6268927708bdc0f2e815b9610bc42d94d)